packages/purpose-process/src/services/purposeService.ts

```diff
@@ -189,6 +191,17 @@ const retrieveRiskAnalysis = (
   return riskAnalysis;
 };
 
+async function retrieveTenantKind(
+  tenantId: TenantId,
+  readModelService: ReadModelService
+): Promise<TenantKind> {
+  const tenant = await retrieveTenant(tenantId, readModelService);
+  if (!tenant.kind) {
+    throw tenantKindNotFound(tenant.id);
+  }
```

Comment:
This could be replaced with `assertTenantKindExists` but this validator is not in this PR

Reply:
@Carminepo2 check [this](https://github.com/pagopa/interop-be-monorepo/pull/476#discussion_r1598410620)